### PR TITLE
Automation service updates

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request adds updates to this article:
https://help.sumologic.com/docs/platform-services/automation-service/automation-service-playbooks/

Issue: Asana ticket "[Update Automation Service documentation per new platform service going live](https://app.asana.com/0/1183741177809740/1205920000503933/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/platform-services/automation-service/automation-service-playbooks.md [hunk 1]
```diff
@@ -43,15 +43,26 @@ The following procedure provides a brief introduction to how to create a playboo
...
+   * [**Action**](#add-an-action-node-to-a-playbook). Automatically take specific actions such as enriching data or taking containment steps.
+   * [**Condition**](#add-a-condition-node-to-a-playbook). Use conditional statements to define what actions should be taken in response to previous inputs.
+   * [**Playbook**](#add-a-playbook-node-to-a-playbook). Call other playbooks in response to conditional statements.
+   * [**Task**](#add-a-task-node-to-a-playbook). Assign a task to an individual.
+   * [**User Choice**](#add-a-user-choice-node-to-a-playbook). Pause playbook execution until a user selects an option. 
+
+
+### Add an action node to a playbook
+
+An action node in a playbook runs an operation. String actions together in the playbook to perform a workflow.
```
Comment (kimsauce) [inline docs/platform-services/automation-service/automation-service-playbooks.md:65]:
```suggestion
An action node in a playbook runs an operation. You can string actions together in the playbook to perform a workflow.  
```

File: docs/platform-services/automation-service/automation-service-playbooks.md [hunk 2]
```diff
@@ -99,4 +120,60 @@ results further by selecting an **AND/OR** operator to define another
 condition.
 1. Click **Update**.
 1. When you create a new condition, you need to define what happens
-when their results meet one of your criteria. Draw lines to nodes to define the flow for success, failure, or other condition options.
\ No newline at end of file
+when their results meet one of your criteria. Draw lines to nodes to define the flow for success, failure, or other condition options.
```
Comment (kimsauce) [inline docs/platform-services/automation-service/automation-service-playbooks.md:123]: looks like an unintentional line break - should be joined with previous line

**Review by kimsauce — APPROVED** [review]
approved pending suggestions

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Pull request merged**